src/Tasks/MigrateNamespaceTask.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace WeDevelop\ElementalMedia\Tasks;
+
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\Dev\BuildTask;
+use WeDevelop\ElementalMedia\Model\ElementalMedia;
+
+class MigrateNamespaceTask extends BuildTask
+{
+    protected $title = 'Elemental media namespace migration';
+
+    protected $description = 'Migrate elemental media TheWebmen namespace to WeDevelop namespace';
+
+    private static $segment = 'migrate-elemental-media-namespace';
+
+    public function run($request)
+    {
+        $elements = BaseElement::get()
+            ->where([
+                'ClassName' => 'TheWebmen\ElementalMedia\Model\ElementalMedia'
+            ]);
+
+        $counter = 0;
+        $totalElements = $elements->count();
+
+        print_r(sprintf("Starting migration of %s elements\n\n", $totalElements));
```

Comment:
Specific reason you use `print_r` and `sprintf` instead of just `printf`?

Reply:
No, I actually copied the code from another module that does the same thing. But wel spotted, I'll fix it.